addon/mixins/dirty-confirm-route.js

```diff
@@ -34,6 +34,9 @@ export default Ember.Mixin.create({
         case ROLLBACK: return true;
         case BUBBLE: return true;
       }
+    },
+    rollbackModel: function(model) {
```

Comment:
Yes, as this doesn't pertain to the route's model.  This pertains to a model that could be anything in regard to a modal.